xml/obs_source_service.xml

```diff
@@ -7,154 +7,299 @@
+   These services just fail in the error case which leads to the build
+   state "broken". Or validation can happen by redoing a certain action and
+   store the result as new file as <command>download_files</command> is doing.
+   In this case the newly generated file will be used instead of the committed
+   one during build. </para>
+ </sect1>
+
+ <sect1 xml:id="sec.obs.sserv.create">
+  <title>Creating Source Service Definitions</title>
+  <para>
+   Source services are definied in the <filename>_service</filename> file
+   and are either part of the package sources or used project-wide.
+   <remark>toms 2017-08-31: Not sure where this _project belongs; where
+   is this _project package?
+   "It is either part of the package sources
+   or used project-wide when stored inside the <filename>_project</filename>
+   package." </remark>
+  </para>
+  <para> The <filename>_service</filename> file contains a list of services
+   which get called in the listed order. Each service may define a list of
+   parameters and a mode.
+   The project wide services get called after the per package defined
+   services.
+   </para>
+  <para>
+   The file is in XML format like this example:
+  </para>
+  <screen>&lt;services&gt;
   &lt;service name="download_files" mode="trylocal" /&gt;
   &lt;service name="verify_file"&gt;
     &lt;param name="file"&gt;krabber-1.0.tar.gz&lt;/param&gt;
     &lt;param name="verifier"&gt;sha256&lt;/param&gt;
     &lt;param name="checksum"&gt;7f535a96a834b31ba2201a90c4d365990785dead92be02d4cf846713be938b78&lt;/param&gt;
   &lt;/service&gt;
   &lt;service name="update_source" mode="disabled" /&gt;
-&lt;/services&gt;
-</screen>
-   <para>This example downloads the files via download_files service via the given
-   URLs from the spec file. When using osc this file gets committed as part of
-   the commit. Afterwards the krabber-1.0.tar.gz file will always be compared
-   with the sha256 checksum. And last but not least there is the
-    <command>update_source</command> service mentioned, which is usually not
-   executed. Except when <command>osc service disabledrun</command> is called,
-   which will try to upgrade the package to a newer source version available
-   online. </para>
+&lt;/services&gt;</screen>
+  <para>
+   With the example above, the following services are executed in the
+   following order:
+  </para>
+  <orderedlist>
+   <listitem>
+    <para>
+     Downloads the file via the <systemitem>download_files</systemitem>
+     service using the URL from the Spec file.
+     When using osc, the downloaded file gets committed as part of the commit.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Compares the downloaded file (<filename>krabber-1.0.tar.gz</filename>)
+     against the SHA256 checksum.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     If enabled, upgrade the source to a newer version available online.
+     This service is usually not executed unless when you run
```

Comment:
remove "usually" (as the only condition when it is run is named in the second half of the sentence)
remove "when"

Reply:
Fixed with the last change from https://github.com/openSUSE/obs-docu/pull/81#discussion_r136389296